terragraph_planner/los/core.py

```diff
@@ -307,7 +307,7 @@ def compute_los(
         )
         for s in sites
     ]
-    exclusion_zones_wkts = [zone.to_wkt() for zone in exclusion_zones]
+    exclusion_zones_wkts = [zone.wkt for zone in exclusion_zones]
```

Comment:
Is this an API change in GeoPandas in later versions? Does this change work for earlier versions we had been using?

Reply:
Does not work in the old version, so I will give a minimal version of shapely.